pom.xml

```diff
@@ -473,7 +473,7 @@
 		<dependency>
 			<groupId>org.jsoup</groupId>
 			<artifactId>jsoup</artifactId>
-			<version>1.14.2</version>
+			<version>1.15.3</version>
```

Comment:
@CalebSLane , am reverting this for now , since it breaks things .
The `org.jsoup.safety.Whitelist` was [removed](https://www.javadoc.io/doc/org.jsoup/jsoup/1.14.2/deprecated-list.html) from the 1.15.3 version which is depended on  by the `org.hibernate.validator`

Reply:
the longer-term solution is to upgrade our` org.hibernate.validator` dependence too